portfolio/src/main/webapp/script.js

```diff
@@ -165,3 +166,13 @@ function createCommentElement(comment) {
 
   return commentElement;
 }
+
+/**
+ * Delete all comments from datastore.
+ */
+function deleteAllComments() {
+  fetch('/delete-data', {
+    method: 'POST',
+  })
```

Comment:
Could this be one line, since the method is so short? Not sure if that's within best practices or not.